pymc3/distributions/distribution.py

```diff
@@ -2,17 +2,21 @@
 import theano.tensor as tt
 from theano import function
 
+from . import transforms
```

Comment:
Tried/failed on my own branch -- it is funny that tests pass on python3, which made me think it is just import order.